packages/firestore/test/unit/local/query_cache.test.ts

```diff
@@ -68,35 +80,35 @@ describe('IndexedDbQueryCache', () => {
   genericQueryCacheTests();
 });
 
+const QUERY_ROOMS = Query.atPath(path('rooms'));
+const QUERY_HALLS = Query.atPath(path('halls'));
+const QUERY_GARAGES = Query.atPath(path('garages'));
```

Comment:
Now that these are file-level globals can you move them to the top of the file (above `persistence` and `cache`)?

Reply:
Undid this change.